lib/archethic/bootstrap.ex

```diff
@@ -209,13 +211,55 @@ defmodule Archethic.Bootstrap do
     end
 
     Archethic.Bootstrap.NetworkConstraints.persist_genesis_address()
+
+    if P2P.authorized_node?() && P2P.available_node?() do
+      resync_network_chain([:oracle, :node_shared_secrets], P2P.authorized_and_available_nodes())
+    end
+
     Sync.publish_end_of_sync()
     SelfRepair.start_scheduler()
 
     :persistent_term.put(:archethic_up, :up)
     Listener.listen()
   end
 
+  @spec resync_network_chain(list(atom), list(P2P.Node.t()) | []) :: :ok
+  def resync_network_chain(_type_list, _nodes = []),
+    do: Logger.info("Enforced Resync of Network Txs: failure, No-Nodes")
+
+  def resync_network_chain(type_list, nodes) when is_list(nodes) do
+    # spawn process for each type of txn to sync
+    Task.Supervisor.async_stream_nolink(Archethic.TaskSupervisor, type_list, fn type ->
+      # acquire a(any) address of type txn, (for querying DB)
+      with addr when is_binary(addr) <- get_addr(type),
+           {:ok, rem_last_addr} <- TransactionChain.resolve_last_address(addr),
+           false <- TransactionChain.get_last_address(addr) |> elem(0) == rem_last_addr,
+           {:ok, tx} <- TransactionChain.fetch_transaction_remotely(rem_last_addr, nodes),
+           :ok <- Replication.validate_and_store_transaction_chain(tx) do
+        Logger.info("Enforced Resync: Success", transaction_type: type)
+      else
+        e when e in [nil, true, []] ->
```

Comment:
When it return true it means that you already have the last address in DB, so you should'nt log an "error" but log that resync is not needed